include/mesh.h

```diff
@@ -50,13 +52,21 @@ class Mesh {
   //! Remove unconnected nodes from the mesh
   void remove_unconnected_nodes();
 
+  //! Return vector of nodal pointers
```

Comment:
Remove these, and make assembler a `friend` class

Reply:
If it is a friend class, then we don't need to even input the mesh. Yes, it will still work for multiple meshes